docs/tutorialkit.dev/src/content/docs/reference/react-components.mdx

```diff
@@ -22,6 +22,13 @@ import sourceUseWebcontainer from "@components/react-examples/hooks/useWebcontai
 
 You can reuse TutorialKit's React components in your own applications. They are designed to work well with the [WebContainer API][webcontainers].
 
+:::note
+Using `@tutorialkit/react` package directly without `@tutorialkit/astro` is **experimental** at the moment.
+This package may introduce breaking changes in patch and minor version updates. Pay extra attention when updating the versions.
+
+We are planning to stabilize component API of `@tutorialkit/react` in future.
+:::
+
```

Comment:
This renders to

<img src="https://github.com/user-attachments/assets/ea9cba66-6fd1-41d3-8ed9-f4293ac48504" width="480px" />